Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -651,36 +660,22 @@ public Task ProcessWebSocketRequest(HttpContext context)
             return _socketListener.ProcessWebSocketRequest(context);
         }
 
-        // this method was left for compatibility with third party clients
-        private static string NormalizeEmbyRoutePath(string path)
+        private string NormalizeEmbyRoutePath(string path)
         {
-            if (path.StartsWith("/", StringComparison.OrdinalIgnoreCase))
-            {
-                return "/emby" + path;
-            }
-
-            return "emby/" + path;
+            _logger.LogDebug("Normalizing /emby route");
+            return _baseUrlPrefix + "/emby" + NormalizeUrlPath(path);
```

Comment:
when `_baseUrlPrefix` is normalized that `/` in `/emby` is no longer needed

Reply:
It is - normalization only adds preceding `/`s by design. Each component must have a leading `/`.